### ISSUE TITLE
Which mikado version to use?

### ISSUE THREAD

**Issue by javbioUMD** [EI-CoreBioinformatics/mikado#299]
Hi, 

I would like try mikado for my species data set but I am having a hard time configuring it and deciding which version of mikado I should use. 

I cloned this repository and installed it running the setup.py and using pip3 (conda installation did not work for me). Initially I thought I downloaded the 1.2.4 version but when I run mikado --version it gives me v2.0rc6. The main issue here is that the documentation/tutorials seems to be based on v1.2.4 which explains why it has been hard to follow the tutorials. 

I guess I could easily just install v1.2.4 but I wonder if this is a good idea since it seems there have been a lot of changes to v2.0rc6.

Any coming documentation based on the latest mikado version?

Best,

Javier

**Comment by lucventurini** [comment]
Dear @javbioUMD 

Apologies for the late reply, in these crazy times I failed to see the notification. 
You are not the only person requesting news about the version to install - @swarbred , @ljyanesm and I are working to get a functional v2 into conda, docker and PyPI as soon as. We plan to probably make a release candidate and upload it for those repositories as soon as #270 and #280 are marked as resolved, which should happen very soon.

Thank you for your patience.

**Comment by lucventurini** [comment]
We now have version 2.0rc2 on both [PyPI](https://pypi.org/project/Mikado/2.0rc2/) and [BioConda](https://anaconda.org/bioconda/mikado). Please use this version and let us know whether there is any other outstanding issue!
